### PR TITLE
Add Trickle Diff Jenkins Job

### PR DESCRIPTION
**Pull request by ncteisen**

### CODE REVIEW

**Comment by matt-kwong** [comment]
I would undo the renaming of `run_performance.sh` since that's what Jenkins tries to run in the performance job. You can put in the change again before you merge and change the performance job config as you merge.

**Comment by matt-kwong** [comment]
To test `run_trickle_diff.sh`, you could temporarily rename it to `run_performance.sh` to make Jenkins run it on this PR. When ready to merge, I'll create a new Jenkins job to run `run_trickle_diff.sh`.

**Comment by ncteisen** [comment]
Good call. Trying that out now

**Comment by ncteisen** [comment]
So I know that this is flaky, however, I would be happy to push this forward to start getting data.

There were 10 failures in the last run, and there were 502 successes. So it's not that flaky.

**Comment by ncteisen** [comment]
TL;DR the python files here are ready for review

**Comment by matt-kwong** [comment]
Can you format the PR comment to use newlines? Right now I have to scroll right a lot to read `Corrupt JSON data` in the comment.

**Comment by ncteisen** [comment]
So the flakes are all caused by my attempts to mock out gpr_now and speed up trickle.

HOWEVER, now that this has its own jenkins job, it can run well, in a timely manner, without the speedup. So I removed all the C++ code from the PR, and will work on that separately. This should be good to go

**Comment by ncteisen** [comment]
Awesome. I'll let it run once more, rename, see the original run, then merge

**Comment by ncteisen** [comment]
Both runs still work. There are no other tests for these scripts, merging early.
